client/js/components/procedure/DpSimplifiedNewStatementForm.vue

```diff
@@ -578,6 +582,11 @@ export default {
   methods: {
     setInitialValues () {
       this.values = { ...this.initValues }
+
+      if (this.values.quickSave) {
+        this.values.text = this.values.quickSave
+      }
+
```

Comment:
isn't this be redundant with DpConvertAnnotatedPdf.vue:213 ?
if not exactly, I think this logic too should only live in the parent component

Reply:
Ah yes indeed, this could be move too, thanks! https://github.com/demos-europe/demosplan-core/pull/3992/commits/e10aa989fe41133723dd986e16ff8d93218b3703